docs/UserGuide.md

```diff
@@ -57,19 +57,31 @@ From here onwards, the term `item` and `delivery` are used specifically for inve
 <div markdown="block" class="alert alert-info">
 **:information_source: Notes about the item command format:**<br>
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add-i n/Chicken`.
+* Commands with suffix of `-i` refer to `item` related commands, while those with suffix of `-d` refer to `delivery` related commands. <br>
+  e.g. `add-i` is used to add an `item` and `add-d` is used to add a `delivery`
+  
+* In the format of commands, `INDEX` refers to the numbering that is displayed.
+  <br> ![Index](images/Index.png)<br>
+
+* In the format of commands, words in `UPPER_CASE` are what you would input.<br>
+  e.g. for a command with format `add-i n/NAME q/QUANTITY`, <br>
+  replace `NAME` and `QUANTITY` with your input: `add-i n/Chicken q/10`.
+
+* In the format of commands, square brackets are to indicate optional fields. The square brackets are not necessary.<br>
```

Comment:
Actually i meant to say that the square brackets themselves are not necessary bc that was what Aileen feedbacked to us

Reply:
Iirc that was because Aileen had a misunderstanding between the command format and the actual command. Saying square brackets are not necessary is wrong because if you put them in the actual command then it's wrong

Reply:
ok fixed in latest commit